File: Code/NewsWebsiteBackEnd/DTO/Article/CreateArticleViewModel.cs
namespace NewsWebsiteBackEnd.DTO.Article
{
    public class CreateArticleViewModel
    {
        public string Title { get; set; }
        public string CoverImagePath { get; set; }
        public string Summary { get; set; }
        public bool IsPublished { get; set; }
        public string? Tags { get; set; }
        public string BodyStructureAsHtmlCode { get; set; }
        public string BodyStructureAsText { get; set; }
        public int CategoryId { get; set; }
    }
}

File: Code/NewsWebsiteFrontEnd/src/app/models/article/create-article-view-model.ts
export interface CreateArticleViewModel {
  title: string;
  coverImagePath: string;
  summary: string;
  isPublished: boolean;
  tags?: any; // in angular it is array of strings but in the api it is a string spletted by comma
  bodyStructureAsHtmlCode: string;
  bodyStructureAsText: string;
  categoryId: number;
  location: string;
}
